src/observer.ts

```diff
@@ -108,7 +109,9 @@ export class DataLayerObserver {
 
     if (rules) {
       rules.forEach((rule: DataLayerRule) => this.registerRule(rule));
+      Logger.getInstance().info(`DLO rule count: ${rules.length}`);
     }
+    Logger.getInstance().info(`DLO constructor time (ms): ${startTime - Date.now()}`);
```

Comment:
Maybe a new `LogMessageType.Initialization` field and passing the `timing` in the context.